### ISSUE TITLE
Stabilize heal tests

### ISSUE THREAD

**Issue by denis-tingaikin** [networkservicemesh/sdk#375]
## Problem statement 

After the latest merges, we faced new unstable heal tests:
```
TestHealClient_Request
TestNewClient_MissingConnectionsInInit
```

## Logs

```
=== Failed
=== FAIL: pkg/networkservice/common/heal TestHealClient_Request (0.07s)
    client_test.go:116: 
        	Error Trace:	client_test.go:116
        	Error:      	Condition never satisfied
        	Test:       	TestHealClient_Request

=== FAIL: pkg/networkservice/common/heal TestNewClient_MissingConnectionsInInit (0.12s)
    client_test.go:217: 
        	Error Trace:	client_test.go:217
        	Error:      	Condition never satisfied
        	Test:       	TestNewClient_MissingConnectionsInInit
```

## Link to CI job

https://github.com/networkservicemesh/sdk/runs/854630760?check_suite_focus=true

**Comment by denis-tingajkin** [comment]
https://github.com/networkservicemesh/sdk/pull/371/checks?check_run_id=854581819

**Comment by denis-tingajkin** [comment]
@haiodo Looks like delay has not helped https://github.com/networkservicemesh/sdk/runs/893129856

**Comment by xzfc** [comment]
`TestHealClient_Request` is being fixed in #493 by @edwarnicke.

I'll take a look at `TestNewClient_MissingConnectionsInInit` which is disabled at the moment.

**Comment by edwarnicke** [comment]
@xzfc Certainly trying to track the issue down :)
